miso-web/src/spring-test/java/uk/ac/bbsrc/tgac/miso/webapp/springtest/AbstractST.java

```diff
@@ -410,4 +413,85 @@ protected ResultActions baseTestGetById(String controllerBase, int id) throws Ex
 
     return result;
   }
+
+  /**
+   * Simple utility method to turn a list of integer ids into a readable string of ids (as read by the
+   * miso.core.util package)
+   */
+  protected String idListString(List<Integer> ids) {
+    String result = "";
+    for (int i = 0; i < ids.size() - 1; i++) {
+      result += ids.get(i) + ",";
+    }
+    result += ids.get(ids.size() - 1); // last id
+    return result;
+  }
+
+  /**
+   * Tests the properties of objects in a JSON array.
+   */
+  private void testJsonArrayProperties(String response, List<HashMap<String, Object>> properties) throws Exception {
+    assertEquals(Integer.valueOf(properties.size()), JsonPath.read(response, "$.length()"));
+    for (int i = 0; i < properties.size(); i++) {
+      HashMap<String, Object> currMap = properties.get(i);
+      for (Map.Entry<String, Object> entry : currMap.entrySet()) {
+        String key = entry.getKey();
+        Object expectedValue = entry.getValue();
+        Object actualValue = JsonPath.read(response, "$[" + i + "]." + key);
+
+        assertEquals(expectedValue, actualValue);
+        if (DEBUG_MODE)
+          System.out.println("\n\nEXPECTED IS: " + expectedValue + ", WHILE ACTUAL IS: " + actualValue + "\n\n");
+      }
+    }
+  }
+
+  /**
+   * Tests model static list endpoints.
+   */
+  protected void testModelList(String url, String listAttribute, List<HashMap<String, Object>> properties)
```

Comment:
Unclear what a model list is. Rename `testStaticListPage`. Similar for other methods - `testBulkEditPage`, etc.

Also unclear what `listAttribute` is. Please expand JavaDoc to include descriptions of the parameters. Same for below methods.